### PR TITLE
updated installation and dead links in README

### PR DESCRIPTION
**Pull request by notPlancha**
There was a dead link in the readme so I updated it, and also added instructions for windows users to install it correctly.
Fell free to edit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,15 +28,17 @@ The turntable inspired by [Netease Music](https://music.163.com) and [Smartisan
 
 1. [install spicetify-cli](https://spicetify.app/docs/getting-started/simple-installation), then follow the [Basic Usage](https://spicetify.app/docs/getting-started/basic-usage)
 
-2. clone the repository, then put **SpotifySpice** and **spotifySpice.js** into the **.config/spicetify/**
+2. clone the repository, then put **SpotifySpice** and **spotifySpice.js** into the spicetify config folder
+* Linux/maxOS: **~/.config/spicetify/**
```
Comment (grasonchan) [inline README.md:32]: Typo, it's macOS.

File: README.md [hunk 2]
```diff
@@ -28,15 +28,17 @@ The turntable inspired by [Netease Music](https://music.163.com) and [Smartisan
...
+* Windows Powershell: **$env:APPDATA/spicetify/Extensions/**
```
Comment (grasonchan) [inline README.md:33]: The path is wrong, you need to delete `/Extensions` here.

File: README.md [hunk 3]
```diff
@@ -28,15 +28,17 @@ The turntable inspired by [Netease Music](https://music.163.com) and [Smartisan
...
 ```shell
 cd spotify-spice
-cp -r SpotifySpice ~/.config/spicetify/Themes
-cp spotifySpice.js ~/.config/spicetify/Extensions
+cp -r SpotifySpice <config-folder-path>/Themes
+cp spotifySpice.js <config-folder-path>/Extensions
 ```
 
-3. select the theme and extensions ([Full App Display](https://spicetify.app/docs/getting-started/extensions#full-app-display))
+3. select the theme and extensions ([Full App Display](https://spicetify.app/docs/advanced-usage/extensions#full-app-display))
```
Comment (grasonchan) [inline README.md:41]: All Spicetify links are dead, can you help update the links (And the Step 1 of the Installation, `then follow the [Basic Usage](https://spicetify.app/docs/getting-started/basic-usage)` is no longer need.) ?

Comment (notPlancha) [reply README.md:41]: should be done

**Comment by notPlancha** [comment]
everything should be right now

**Comment by grasonchan** [comment]
close #12

**Comment by grasonchan** [comment]
@notPlancha Thanks for your contribution.

**Pull request merged**